dev/src/DocFx/Node/VisibilityTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Dev\DocFx\Node;
+
+trait VisibilityTrait
+{
+    public function isPublic(): bool
+    {
+        return 'public' === (string) $this->xmlNode['visibility'];
+    }
+
+    public function isInherited(): bool
+    {
+        if ($this->xmlNode->inherited_from) {
+            return true;
+        }
+
+        return false;
```

Comment:
return $this->xmlNode->inherited_from
```suggestion
        return $this->xmlNode->inherited_from;
```